ui/common/css/vendor/chessground/_coords-colors.scss

```diff
@@ -94,3 +94,51 @@ coords {
     color: var(--cg-coord-color-amazons-p2);
   }
 }
+
+.variant-breakthroughtroyka {
+  coords {
+    text-shadow: var(--cg-coord-shadow-breakthroughtroyka);
+  }
+
+  .orientation-p1 .files coord:nth-child(2n + 1),
+  .orientation-p1 .ranks coord:nth-child(2n),
+  .orientation-p2 .files coord:nth-child(2n),
+  .orientation-p2 .ranks coord:nth-child(2n + 1),
+  .orientation-right .files coord:nth-child(2n + 1),
+  .orientation-right .ranks coord:nth-child(2n + 1),
+  .orientation-left .files coord:nth-child(2n + 1),
+  .orientation-left .ranks coord:nth-child(2n + 1) {
+    color: var(--cg-coord-color-breakthroughtroyka-p1);
+  }
+
+  .orientation-p1 .files coord:nth-child(2n),
+  .orientation-p1 .ranks coord:nth-child(2n + 1),
+  .orientation-p2 .files coord:nth-child(2n + 1),
+  .orientation-p2 .ranks coord:nth-child(2n),
+  .orientation-right .files coord:nth-child(2n),
```

Comment:
I do not think we need to keep these orientation-<left|right> ?

Reply:
that is for lines of action, I don't think left is used but there for completion.

Reply:
but yes you are right we just need p1/p2 orientation for breakthrough.